src/main/java/uk/gov/justice/digital/delius/data/api/Sentence.java

```diff
@@ -30,4 +30,6 @@
     private LocalDate startDate;
     private LocalDate terminationDate;
     private String terminationReason;
+    @ApiModelProperty(value = "Type of sentence")
+    private String sentenceType;
```

Comment:
I would have thought maybe a KeyValue might be better so you get the description and code rather than just the code

Reply:
Good call, I have done so